homeassistant/components/ring/__init__.py

```diff
@@ -99,10 +60,8 @@ def token_updater(token):
     async def async_refresh_all(_: ServiceCall) -> None:
         """Refresh all ring data."""
         for info in hass.data[DOMAIN].values():
-            await info["device_data"].async_refresh_all()
-            await info["dings_data"].async_refresh_all()
-            await hass.async_add_executor_job(info["history_data"].refresh_all)
-            await hass.async_add_executor_job(info["health_data"].refresh_all)
+            await info[RING_DEVICES_COORDINATOR].async_refresh()
+            await info[RING_NOTIFICATIONS_COORDINATOR].async_refresh()
 
     # register service
     hass.services.async_register(DOMAIN, "update", async_refresh_all)
```

Comment:
Service removed and `homeassistant.update_entity` service added to tests

Reply:
You were too fast :) Instead of removing we should deprecate the service for six releases and create a repair issue. So users have time to move to the other service.

The best would be to deprecate the service in a follow-up so we keep this PR small.
Sorry for the inconvenience. I wanted to ask why we need this update service.

Reply:
Ok deprecation is now in draft PR https://github.com/home-assistant/core/pull/108781.  This comment should be fully addressed.